tests/plugins/test_pin_operator_digests.py

```diff
@@ -511,6 +511,59 @@ def test_orchestrator(self, docker_tasker, tmpdir, caplog):
         }
         assert self._get_worker_arg(runner.workflow) == replacement_pullspecs
 
+        expected_result = {
+            'related_images': {
+                'pullspecs': [
+                    {
+                        'original': ImageName.parse('old-registry/ns/spam:1'),
+                        'new': ImageName.parse('new-registry/new-ns/new-spam@sha256:4'),
+                        'pinned': True,
+                        'replaced': True
+                    }, {
+                        'original': ImageName.parse('old-registry/ns/spam@sha256:4'),
+                        'new': ImageName.parse('new-registry/new-ns/new-spam@sha256:4'),
+                        'pinned': False,
+                        'replaced': True
+                    }, {
+                        'original': ImageName.parse('private-registry/ns/baz:1'),
+                        'new': ImageName.parse('public-registry/ns/baz@sha256:3'),
+                        'pinned': True,
+                        'replaced': True
+                    }, {
+                        'original': ImageName.parse('private-registry/ns/baz@sha256:3'),
+                        'new': ImageName.parse('public-registry/ns/baz@sha256:3'),
+                        'pinned': False,
+                        'replaced': True
+                    }, {
+                        'original': ImageName.parse('registry.private.example.com/ns/foo:1'),
+                        'new': ImageName.parse('registry.private.example.com/ns/foo@sha256:1'),
+                        'pinned': True,
+                        'replaced': True
+                    }, {
+                        'original': ImageName.parse('registry.private.example.com/ns/foo@sha256:1'),
+                        'new': ImageName.parse('registry.private.example.com/ns/foo@sha256:1'),
+                        'pinned': False,
+                        'replaced': False
+                    }, {
+                        'original': ImageName.parse('weird-registry/ns/bar:1'),
+                        'new': ImageName(
+                            registry='weird-registry', repo='new-bar', tag='sha256:2'),
+                        'pinned': True,
+                        'replaced': True
+                    }, {
+                        'original': ImageName.parse('weird-registry/ns/bar@sha256:2'),
+                        'new': ImageName(
+                            registry='weird-registry', repo='new-bar', tag='sha256:2'),
+                        'pinned': False,
+                        'replaced': True
+                    },
+                ],
+                'created_by_osbs': True,
+            }
+        }
+
+        assert result['pin_operator_digest'] == expected_result
```

Comment:
Can we enhance `test_orchestrator_no_pullspecs` to verify the result is `None` ? right now it only verifies that arguments were not forwarded to the worker.